c2corg_api/models/waypoint.py

```diff
@@ -139,7 +139,7 @@ class _WaypointMixin(object):
     phone_custodian = Column(String(50))
 
     # gardiennage (gite, camp_site, hut, abri, bivouac, base_camp)
-    custodianship = Column(Boolean)
+    custodianship = Column(enums.custodianship_type)
```

Comment:
I meant it is a field with more value than yes/no. But only one value at a time can be selected. Do you think I should use another word than "multi-value"?